tests/FSharp.Configuration.Tests/FSharp.Configuration.Tests.fsproj

```diff
@@ -30,21 +31,8 @@
     </Content>
     <Content Include="TextFile.txt" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
-    <Reference Include="FSharp.Configuration">
-      <HintPath>..\..\bin\lib\net45\FSharp.Configuration.dll</HintPath>
-    </Reference>
-    <Reference Include="YamlDotNet">
-      <HintPath>..\..\bin\lib\net45\YamlDotNet.dll</HintPath>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
-    <Reference Include="FSharp.Configuration">
-      <HintPath>..\..\bin\lib\netstandard2.0\FSharp.Configuration.dll</HintPath>
-    </Reference>
-    <Reference Include="YamlDotNet">
-      <HintPath>..\..\bin\lib\netstandard2.0\YamlDotNet.dll</HintPath>
-    </Reference>
+  <ItemGroup>
+    <ProjectReference Include="../../src/FSharp.Configuration/FSharp.Configuration.fsproj" />
```

Comment:
Could build fails because of this change? @baronfel

Reply:
I don't think so, because this is the reference to FSharp.Configuration, and that `-r` reference for FSharp.Configuration seems to be showing up correctly in the build command lines to FSC.  maybe, though?